WordPressSharedTests/NSDateHelperTest.swift

```diff
@@ -33,22 +33,22 @@ class NSDateHelperTest: XCTestCase {
         XCTAssertEqual(components.month, data.month)
         XCTAssertEqual(components.day, data.day)
     }
-    
+
     /// Verifies that `mediumString` produces relative format strings when less than 7 days have elapsed.
-    func testMediumStringRelativeString() {
+    func testToMediumStringRelativeString() {
         let date = Date()
-        XCTAssertEqual(date.mediumString(), "just now")
-        XCTAssertEqual(date.addingTimeInterval(-60*5).mediumString(), "5 minutes ago")
-        XCTAssertEqual(date.addingTimeInterval(-60*60*2).mediumString(), "2 hours ago")
-        XCTAssertEqual(date.addingTimeInterval(-60*60*24).mediumString(), "1 day ago")
-        XCTAssertEqual(date.addingTimeInterval(-60*60*24*6).mediumString(), "6 days ago")
+        XCTAssertEqual(date.toMediumString(), "now")
+        XCTAssertEqual(date.addingTimeInterval(-60*5).toMediumString(), "5 minutes ago")
+        XCTAssertEqual(date.addingTimeInterval(-60*60*2).toMediumString(), "2 hours ago")
+        XCTAssertEqual(date.addingTimeInterval(-60*60*24).toMediumString(), "yesterday")
+        XCTAssertEqual(date.addingTimeInterval(-60*60*24*6).toMediumString(), "6 days ago")
```

Comment:
I wonder if we should add future dates in those tests ("tomorrow", "in 2 days", etc). Even if those are rare, I can see stuff like "scheduled publishing dates" being such cases that we could still encounter.

Reply:
Sure – doesn't hurt!

Addressed in dc42ef1a406a462186ae44cd4b97fc2be8fe4424.